lib/core/src/Cardano/Wallet/Primitive/CoinSelection.hs

```diff
@@ -192,20 +305,20 @@ data SelectionParams = SelectionParams
     }
     deriving (Eq, Generic, Show)
 
--- | Indicates that an error occurred while performing a coin selection.
---
-data SelectionError
-    = SelectionBalanceError Balance.SelectionError
-    | SelectionOutputsError ErrPrepareOutputs
+-- | Indicates that coin selection failed, or a precondition to coin selection
+-- failed.
+data ErrWalletSelection
+    = ErrWalletSelectionBalance SelectionError
+    | ErrWalletSelectionOutputs ErrPrepareOutputs
     deriving (Eq, Show)
 
 -- | Prepares the given user-specified outputs, ensuring that they are valid.
 --
-prepareOutputs
+prepareOutputsForMinUTxO
     :: SelectionConstraints
     -> NonEmpty TxOut
     -> Either ErrPrepareOutputs (NonEmpty TxOut)
-prepareOutputs constraints outputsUnprepared
+prepareOutputsForMinUTxO constraints outputsUnprepared
```

Comment:
Hi @rvl. I think this name change is actually unnecessary, as this function is not just about the minimum UTxO: it also:
- checks the serialized sizes of outputs
- checks that outputs don't have quantities in excess of the maximum (maxBound :: Word64).

I reckon it would be better to leave the name alone here.